protocol/x/assets/lib/lib.go

```diff
@@ -0,0 +1,41 @@
+package lib
+
+import (
+	"math/big"
+
+	"github.com/dydxprotocol/v4-chain/protocol/lib/margin"
+	"github.com/dydxprotocol/v4-chain/protocol/x/assets/types"
+)
+
+// GetNetCollateralAndMarginRequirements returns the net collateral, initial margin, and maintenance margin
+// that a given position (quantums) for a given assetId contributes to an account.
+func GetNetCollateralAndMarginRequirements(
```

Comment:
Naming for this function seems a bit odd.
IMR and MMR are always returned as zero which makes no sense.

Reply:
cc: @jayy04